sdk/scheduler/src/main/java/com/mesosphere/sdk/offer/MesosResourcePool.java

```diff
@@ -154,10 +163,14 @@ public Offer getOffer() {
             if (atomicResources == null) {
                 LOGGER.info("Offer lacks any atomic resources named {}", resourceName);
             } else {
+                String desired = spec.getValue().toString();
```

Comment:
This could use `TextFormat.shortDebugString(spec.getValue())` to avoid newlines in the log